docs/v1/index.bs

```diff
@@ -2050,6 +2050,36 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 </section>
 
 <section>
+  <h2 id="iana-considerations">IANA Considerations</h2>
+  
+  <p>The following registrations are for community review and will be submitted to the <a href="https://www.ietf.org/iesg/">IESG</a> for review, approval, and registration with <a href="https://www.iana.org/">IANA</a>.</p>
```

Comment:
Will this part of the section need to be changed or replaced when the fields are registered to IANA?